src/modules/site-v2/templates/_includes/breadcrumb.html

```diff
@@ -11,7 +11,7 @@
             {% elif alt_parent_breadcrumb %}
             <li class="breadcrumb-item">{{ alt_parent_breadcrumb["title"] }}</li>
             {% elif tool_alt_parent_breadcrumb %}
-                {% if tool_alt_parent_breadcrumb["title"] == 'Strain Catalog' %}
+                {% if tool_alt_parent_breadcrumb["title"] == 'Strain Catalog' or  tool_alt_parent_breadcrumb["title"] == 'Submit Trait'%}
```

Comment:
this code works, but it is brittle if we tweak the wording of the title. Do we have a better internal `id` that we can reference? Something that wouldn't need to change if we tweak the wording of the title.

Reply:
We don't have any IDs for titles, but I'll see if I can tweak it with `request.blueprint`, which would be more reliable.

Reply:
Pushed in the commit 15798ce7eb7b7ad089651dc5731f1e0e03546727